doc/getting_started/getting_started.rst

```diff
@@ -92,10 +92,10 @@ To build an example application follow these steps:
 
 
       # On Linux/macOS
-      export ZEPHYR_GCC_VARIANT=zephyr
+      export ZEPHYR_TOOLCHAIN_VARIANT=zephyr
       export ZEPHYR_SDK_INSTALL_DIR=<sdk installation directory>
       # On Windows
-      set ZEPHYR_GCC_VARIANT=zephyr
+      set ZEPHYR_TOOLCHAIN_VARIANT=zephyr
       set ZEPHYR_SDK_INSTALL_DIR=<sdk installation directory>
 
```

Comment:
I'm thinking it would be good to add here something like:

    .. note:: In previous releases of Zephyr, the ``ZEPHYR_TOOLCHAIN_VARIANT``
       variable was called ``ZEPHYR_GCC_VARIANT``.